### ISSUE TITLE
Eliminate "globId" argument from create_page.R?

### ISSUE THREAD

**Issue by ktrutmann** [ndphillips/ShinyPsych#67]
As I understand it a pageList will always contain a globId variable.
Can we not just extract it from the pageList with pageList$globId?
What is the purpose of having an additional globId argument in the createPage function?

If there is none and it is an oversight, I could rewrite the code to extract the Id from the pageList, but leave the argument in (for now) until we elminated all its uses.

**Comment by mdsteiner** [comment]
yes I think it is not really necessary and could be deleted. I will double check (I hope I'll soon be done with the conditional feature and will then anyway have to go through the code to make sure I didn't break something) 
But I think I would still keep it as an optional argument for backward compatibility
